jbot/src/main/java/me/ramswaroop/jbot/core/slack/models/User.java

```diff
@@ -9,6 +9,8 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class User {
     private String id;
+    @JsonProperty("team_id")
+    private String teamId;    
```

Comment:
Sorry, but I couldn't find `team_id`. The [user_change](https://api.slack.com/events/user_change) event returns an `User` object and `User` object didn't have a property called `team_id`.